pipeline/metadata/schema.py

```diff
@@ -0,0 +1,402 @@
+"""Schema for various data types passed around in the pipeline and bigquery"""
+
+from __future__ import absolute_import
+from __future__ import annotations  # required to use class as a type inside the class
+
+import dataclasses
+from dataclasses import dataclass
+from typing import Optional, List, Dict, Any, Union
+
+# pylint: disable=too-many-instance-attributes
+
+
+@dataclass
+class SatelliteAnswer():
+  """Class for keeping track of Satellite answer content"""
+  # Keys
+  ip: str
+  # Metadata
+  asnum: Optional[int] = None
+  asname: Optional[str] = None
+  http: Optional[str] = None
+  cert: Optional[str] = None
+  matches_control: Optional[str] = None
+
+
+@dataclass
+class SatelliteAnswerMetadata(SatelliteAnswer):
+  """Satellite Answer Metadata.
+
+  When this metadata is being passed around
+  it needs an additional date field to keep track of when it's valid.
+  In the final SatelliteAnswer object written to bigquery
+  we don't include that field since it's redundant.
+  """
+  date: str = ''
+
+
+def merge_satellite_tags(answer: SatelliteAnswer,
```

Comment:
Pass two `AnnotatedSatelliteAnswer` objects instead.
You can get the second parameter from the DatedAnnotatedSatelliteAnswer with `dasa.answer`.